pallets/gear-bank/src/lib.rs

```diff
@@ -521,7 +555,15 @@ pub mod pallet {
             // `*_no_transfer` function above.
             //
             // This call does only currency trait final transfer.
-            Self::withdraw(destination, value).unwrap_or_else(|e| unreachable!("qed above: {e:?}"));
+            Self::withdraw(destination, value).unwrap_or_else(|e| {
+                let err_msg = format!(
+                    "Pallet::transfer_value: withdraw failed. \
+                    Account id - {account_id:?}, destination: {destination:?}, value - {value:?}. Got error - {e:?}"
```

Comment:
same, but destination account balance should be included

Reply:
Fixed